nav2_system_tests/worlds/turtlebot3_ros2_demo_obstacle.world

```diff
@@ -83,7 +83,7 @@
           <pose>-0.032 0 0 0 0 0</pose>
           <geometry>
             <mesh>
-              <uri>model://turtlebot3_burger/meshes/burger_base.dae</uri>
+              <uri>model://turtlebot3_waffle/meshes/waffle_base.dae</uri>
```

Comment:
These changes to the world files are only changing the visual elements associated with the model. All the sizes and weights used by the physics part of the engine are still the Burger model, aren't they?

I think the important part are the physical characteristics.

Reply:
I suppose I misinterpreted the ticket. I thought all the params were for waffle but just had burger visuals. 

Will update